apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -58,18 +60,19 @@ function ValidatorPageResult(): JSX.Element {
         return apys.length > 0 ? roundFloat(averageAPY / apys.length) : 0;
     }, [validatorsApy]);
 
-    const lastEpochRewardOnAllValidators = useMemo(() => {
-        if (!validatorEvents) return null;
-        let totalRewards = 0;
-
-        validatorEvents.forEach(({ parsedJson }) => {
-            totalRewards += Number(
-                (parsedJson as { pool_staking_reward: string }).pool_staking_reward,
-            );
-        });
-
-        return totalRewards;
-    }, [validatorEvents]);
+    const enhancedRpc = useEnhancedRpcClient();
+    const { data: epochData } = useQuery({
+        queryKey: ['epoch', data?.epoch],
+        queryFn: async () => {
+            const epoch = Number(data?.epoch || 0);
+            return enhancedRpc.getEpochs({
+                cursor: epoch === 0 || epoch === 1 ? undefined : (epoch - 2).toString(),
```

Comment:
we might benefit from a comment here explaining why -2, just for the future us, now is clear but maybe in 3 months isnt 🙈 
Also, epoch I presume will never be 0 because the current epoch at minimum is always epoch 1, but thats ok, maybe we could just make `epoch < 2 ? undefined : (epoch - 2).toString()`